packages/playground/src/views/minting_view.vue

```diff
@@ -0,0 +1,217 @@
+          <v-row>
+            <v-col cols="2" class="column-style">
+              <v-list-item v-for="header in fixedHeader" :key="header">
+                {{ header }}
+              </v-list-item>
+            </v-col>
+            <v-col cols="10">
+              <v-list-item> {{ item.node_id }}</v-list-item>
+              <v-list-item> {{ item.farm_id }}</v-list-item>
+            </v-col>
+          </v-row>
+        </v-list>
+      </v-card>
+
+      <v-card class="mt-3">
+        <v-card-title class="font-weight-bold bg-primary" color="primary">Node Resources</v-card-title>
+        <v-list class="custom-list">
+          <v-row>
+            <v-col cols="2" class="column-style">
+              <v-list-item v-for="header in resourcesFHeader" :key="header">
+                {{ header }}
+              </v-list-item>
+            </v-col>
+            <v-col cols="10">
+              <v-list-item>
+                {{ item.cloud_units.cu.toFixed(2) }} | {{ item.correct_cloud_units.cu.toFixed(2) }} |
+                {{ item.fixup_cloud_units.cu.toFixed(2) }}</v-list-item
+              >
+              <v-list-item>
+                {{ item.cloud_units.su.toFixed(2) }} | {{ item.correct_cloud_units.su.toFixed(2) }} |
+                {{ item.fixup_cloud_units.su.toFixed(2) }}</v-list-item
+              >
+              <v-list-item>
+                {{ item.cloud_units.nu.toFixed(2) }} | {{ item.correct_cloud_units.nu.toFixed(2) }} |
+                {{ item.fixup_cloud_units.nu.toFixed(2) }}</v-list-item
+              >
+            </v-col>
+          </v-row>
+        </v-list>
+      </v-card>
+      <v-card class="mt-3">
+        <v-card-title class="font-weight-bold bg-primary">Payout Info</v-card-title>
+        <v-list class="custom-list">
+          <v-row>
+            <v-col cols="2" class="column-style">
+              <v-list-item v-for="header in payoutFHeader" :key="header">
+                {{ header }}
+              </v-list-item>
+            </v-col>
+            <v-col cols="10">
+              <v-list-item> {{ item.minted_tft.toFixed(7) }} TFT</v-list-item>
+              <v-list-item> {{ item.correct_tft.toFixed(7) }} TFT</v-list-item>
+              <v-list-item> {{ item.fixup_tft.toFixed(7) }} TFT</v-list-item>
+              <v-list-item> {{ item.stellar_payout_address }}</v-list-item>
+            </v-col>
+          </v-row>
+        </v-list>
+      </v-card>
+    </v-container>
+  </view-layout>
+</template>
+
+<script lang="ts" setup>
+import { ref } from "vue";
+
+import { useInputRef } from "@/hooks/input_validator";
+
+import { getMintingData } from "../utils/mintings";
+const receiptHash = ref();
+const isValidForm = ref(false);
+const textInput = useInputRef();
+const loading = ref(false);
+const item = ref();
+const infoHeader = ref(["ID", "Farm", "Measured Uptime"]);
+const fixedHeader = ref(["ID", "Farm"]);
+const resourcesHeader = ref(["CU", "SU", "NU", "Cru", "Mru", "Sru", "Hru"]);
```

Comment:
please use better names for vars.
maybe `infoHeader` -> `mintNodeInfoHeader`
`fixedHeader` -> `fixupNodeInfoHeader`
and so on...